int16 pressure
uint16 distance
geometry_msgs/Pose pose